spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -186,7 +189,9 @@
         "type": "personal",
```

Comment:
This is a good place to see what I had thought the expected structure should be. Let me know if I got something wrong here, I definitely got a little confused myself with all the brackets and what not.